packaging/debian/dotnet-debian_config.json

```diff
@@ -44,6 +44,7 @@
         "bin/dotnet-compile-csc" : "usr/bin/dotnet-compile-csc",
         "bin/dotnet-compile-fsc" : "usr/bin/dotnet-compile-fsc",
         "bin/dotnet-compile-native" : "/usr/bin/dotnet-compile-native",
+        "bin/dotnet-mcg": "usr/bin/dotnet-mcg",
```

Comment:
This also needs to be added to 

packaging/debian/dotnet-nightly-debian_config.json